e2e/test/scenarios/joins/reproductions/29795-native-query-join-question-misses-metadata.cy.spec.js

```diff
@@ -0,0 +1,43 @@
+import {
+  restore,
+  visualize,
+  popover,
+  openOrdersTable,
+} from "e2e/support/helpers";
+
+describe("issue 29795", () => {
+  beforeEach(() => {
+    restore();
+    cy.signInAsAdmin();
+  });
+
+  it.skip("should allow join based on native query (metabase#29795)", () => {
+    const NATIVE_QUESTION = "native question";
+    cy.createNativeQuestion({
+      name: NATIVE_QUESTION,
+      native: { query: `Select * FROM "PUBLIC"."ORDERS"` },
+    });
```

Comment:
A few observations:
1. we usually need to visit the native question created through API in order to generate its `result_metadata`
2. to speed things up a bit, please add limit to the query `limit: 5` should work

To achieve 1. you can sue this:
`cy.createNativeQuestion(questionDetails, { visitQuestion: true });` or `cy.createNativeQuestion(questionDetails, { loadMetadata: true });` (which is doing the same thing, but is semantically more correct)

Reply:
added both, thanks!